### PR TITLE
Feature/migrate mobile rule

### PR DESCRIPTION
**Pull request by jycr**

### CODE REVIEW

**Review by dedece35 — DISMISSED** [review]

**Review by utarwyn — COMMENTED** [review]
In this configuration, a lot of mobile rules will be in the "Java part" of the rules specification artifact. So they will be included in both Java and Android plugins. I imagine you have this in mind? as it stands, it's not a problem, but we're going to include some useless rules in the JAR files

Just a thought, open to discussion

**Comment by jycr** [comment]
> In this configuration, a lot of mobile rules will be in the "Java part" of the rules specification artifact. So they will be included in both Java and Android plugins. I imagine you have this in mind? as it stands, it's not a problem, but we're going to include some useless rules in the JAR files
> 
> Just a thought, open to discussion

You're absolutely right.
This is a known consequence which has already been mentioned.
I'm waiting for our ideas to be clearer regarding our "target/platform" (compared to the different plugins proposed by the green-code-initiative community) in order to propose something even more optimal.

**Comment by jycr** [comment]
@jhertout : Do we merge this PR to `main` and make a release to Maven Central? Or do you prefer to wait a little longer? (maybe you want to make fixes?)

**Review by dedece35 — APPROVED** [review]

File: ecocode-rules-specifications/src/main/rules/EC544/xml/EC544.asciidoc [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Score ecoCode: 2sur5
```
Comment (utarwyn) [inline ecocode-rules-specifications/src/main/rules/EC544/xml/EC544.asciidoc:1]: I wonder why this wording is in French?

**Review by utarwyn — APPROVED** [review]
LGTM otherwise 💯 
We can start with this architecture first and improve it if we found a better idea in the future

**Comment by utarwyn** [comment]
@jycr Thank you for the version 0.0.5!
I'm waiting for a new one with rules from #226 (integrated few minutes ago) and rules from this PR if you want

**Pull request merged**